app/controllers/ems_container_controller.rb

```diff
@@ -32,6 +32,11 @@ def ems_container_form_fields
     ems_form_fields
   end
 
+  def new
+    super
+    @ems_types -= [["Atomic", "atomic"], ["Atomic Enterprise", "atomic_enterprise"]]
```

Comment:
oh, you overlooked the small `-` :) Its shorthand for
`@ems_types = @ems_types - []`